src/Package/Impl/Browsers/WebBrowserServices.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.VisualStudio.R.Package.Shell;
 using Microsoft.VisualStudio.R.Packages.R;
 using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.Windows.Core.OS;
```

Comment:
I kept same namespace in Windows and common i.e. There is no 'using Microsoft.R.Editor.Windows`.